Formula/mingw-w64.rb

```diff
@@ -31,9 +31,9 @@ class MingwW64 < Formula
   depends_on "mpfr"
 
   resource "binutils" do
-    url "https://ftp.gnu.org/gnu/binutils/binutils-2.39.tar.xz"
-    mirror "https://ftpmirror.gnu.org/binutils/binutils-2.39.tar.xz"
-    sha256 "645c25f563b8adc0a81dbd6a41cffbf4d37083a382e02d5d3df4f65c09516d00"
+    url "https://ftp.gnu.org/gnu/binutils/binutils-2.40.tar.xz"
+    mirror "https://ftpmirror.gnu.org/binutils/binutils-2.40.tar.xz"
+    sha256 "0f8a4c272d7f17f369ded10a4aca28b8e304828e95526da482b0ccc4dfc9d8e1"
   end
```

Comment:
instead doing a resource, maybe adding as dependency?

Reply:
And if so, that could probably also be the case for `gcc`, just below.

Reply:
yeah, I believe gcc should be a depends_on as well, asking about this. :)

Reply:
That’s not possible as GNU toolchains don’t support cross-targets unlike an LLVM based toolchain.

We’re forced to compile each component for the desired cross-target.

Reply:
👍